workshops/chat/README.md

```diff
@@ -0,0 +1,258 @@
+# Chat
+
+Short link to this workshop: [https://workshops.hackclub.com/chat](https://workshops.hackclub.com/chat)
+
+---
+
+In this workshop we'll be implementing a real-time chat using Firebase. This workshop is a revival of the nowdefunct tutorial made by Firebase itself.
+
+**Table of Contents**
+
+- Part I: Set-up
+- Part II: The HTML File
+- Part III: The JS File
+- Part IV: Publishing and Sharing (and Chatting!)
+- Part V: Hacking
+
+## Part I: Set-up
+
+- Open up your C9 workspace
+- Make a new folder and name it `chat`
+- Make a new file within `chat` and name it `index.html`
```

Comment:
Let's rephrase this to "Make a new file called `index.html` in the `chat` folder". Makes it clearer what the file should be called.